<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="custom_hair">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>커스텀 헤어스타일 적용</string>
    </property>
   </widget>
   <widget class="QPushButton" name="custom_dye">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>커스텀 염색 적용</string>
    </property>
   </widget>
   <widget class="QPushButton" name="preset">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>프리셋 헤어스타일 적용</string>
    </property>
   </widget>
   <widget class="QFrame" name="img_review">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>169</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Made by Master Carry</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hair Ganzi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
